packages/charts/react-charting/src/components/DeclarativeChart/__snapshots__/DeclarativeChartRTL.test.tsx.snap

```diff
@@ -4661,366 +4661,6 @@ exports[`DeclarativeChart Should render heatmapchart in DeclarativeChart 1`] = `
             d="M44.5,0.5H-19.5"
             stroke="currentColor"
           />
-          <g
-            class="tick"
-            opacity="1"
-            transform="translate(42.36963036963037,0)"
-          >
```

Comment:
why did this block get removed

Reply:
This is because containerWidth is 0 in the test environment. Let me mock its value.